Tests/Cell.py

```diff
@@ -1743,6 +1751,7 @@ def test_clear_with_mdx_happy_case(self):
 
         self.tm1.cells.clear_with_mdx(cube=CUBE_NAME, mdx=mdx)
 
+        # this may need looking at
         value = next(self.tm1.cells.execute_mdx_values(mdx=mdx))
```

Comment:
I suppose here we would need to write
``` python
value = self.tm1.cells.execute_mdx_values(mdx=mdx)[0]
```
Don't you think?

Reply:
Yeah, that should work, I left it alone because I didn't have an 11.7 server to test it with.

Reply:
I see. I will accept the MR and take of these with my local 11.7 instance.